docs/application/dotnet/guides/security/dpm.md

```diff
@@ -0,0 +1,99 @@
+
+The main features of the [Tizen.Security.DevicePolicyManager](https://samsung.github.io/TizenFX/master/api/Tizen.Security.DevicePolicyManager.html) namespace are:
+
+- Managing policies
+
+  You can [track the state between the device admin client and the device policy manager](#client_application) using the `DevicePolicyManager` class provided by [Tizen.Security.DevicePolicyManager](https://samsung.github.io/TizenFX/master/api/Tizen.Security.DevicePolicyManager.html).
+
+- Checking restrictions
+
+  You can [check the restriction states of the device](#client_application), such as camera, microphone, Wi-Fi, Bluetooth, and USB, using the properties of the policy classes. [Tizen.Security.DevicePolicyManager](https://samsung.github.io/TizenFX/master/api/Tizen.Security.DevicePolicyManager.html) provides these policy classes.
+
+The following figure illustrates the DPM framework process:
+
+**Figure: DPM framework process**
+
+![DPM framework process](./media/dpm-framework.png)
+
+## Prerequisites
+
+To use the methods and properties of [Tizen.Security.DevicePolicyManager](https://samsung.github.io/TizenFX/master/api/Tizen.Security.DevicePolicyManager.html), include the following in your application:
+
+```
+using Tizen.Security.DevicePolicyManager;
+```
+
+<a name="client_application"></a>
+## Managing Device Policies
+
+To manage device policies:
+
+1. Create a `DevicePolicyManager` instance:
+
+   ```
+       DevicePolicyManager dpm = new DevicePolicyManager();
+   ```
+
+2. Get the policy instance using `getPolicy<T>()` of the DevicePolicyManager instance:
+   ```
+       MediaPolicy mediaPolicy = dpm.getPolicy<MediaPolicy>();
+   ```
+   > **Note**
+   >
+   > The DevicePolicyManager instance must exist when you use the policy instance.
+
+3. Register an event handler to manage the policies of the policy instance:
+
+   ```
+   try
+   {
+       DevicePolicyManager dpm = new DevicePolicyManager();
+       MediaPolicy mediaPolicy = dpm.getPolicy<MediaPolicy>();
+
+       /// Register the event handler
+       mediaPolicy.CameraPolicyChanged += onCameraPolicyChanged;
+   }
+   catch (Exception e)
+   {
+       /// Handle exception
+   }
+
+   /// Create the event handler
+   void onCameraPolicyChanged(Object Sender, PolicyChangedEventArgs args)
+   {
+       Console.WriteLine("PolicyName: " + args.PolicyName + ", Current policy state: " + args.IsAllowed);
+   }
+   ```
+
+4. Check the device restriction state:
+
+   ```
+   /// Check the restriction state of the camera
+   /// false: using the camera is not allowed
+   /// true: using the camera is allowed
+   bool cameraPolicyState = mediaPolicy.IsCameraAllowed;
+   ```
+
+5. When no longer needed, destroy the policy instance and DPM instance:
+
+   ```
+   mediaPolicy.Dispose();
+   dpm.Dispose();
+   ```
+   > **Note**
+   >
+   > The policy instance must be destroyed before the DPM instance.
+
+## Related Information
+- Dependencies
+  - Tizen 5.5 and higher for Mobile
```

Comment:
Done.

Reply:
@mijin85cho 
Sorry for the inconvenience. The comment was discussed by the person, but the appropriate information was not shared across the team. 
I will ensure that the information is spread across the team.
How about changing it to "Tizen 5.5 and higher". 

https://github.com/Samsung/tizen-docs/pull/590#discussion_r262003981
This was one of my previous comment. For a list item, I would like to use sentence case.
Please advice me so that I can ensure consistency in other reviews.

Reply:
@Reni-90 actually this "Dependencies" section not a part of the content.

It is originated form ToC box on Tizen Developer Site.

[Below is one of the TD page](https://developer.tizen.org/development/guides/.net-application/security/privilege-information), as the sample. 

![image](https://user-images.githubusercontent.com/9958374/54014931-8c4f1f80-41c1-11e9-84c6-8c949fb923b3.png)

You can see the red box above. It is not a content, but a mark showing the supported version for each modules in every guide pages..
Please don't use sentence case here.

Reply:
OK
Thank you very much for the explanation.